dockyard/controllers/v1/container.py

```diff
@@ -0,0 +1,210 @@
+from pecan import expose, abort, request
+
+from dockyard.common.container import container 
+
+class Container(object):
+    def __init__(self):
+        self.container = container.Container()
+
+    @expose(generic=True)
+    def index(self):
+        abort(404)
+
+    @index.when(method="DELETE")
+    def delete(self, _id, **kwargs):
```

Comment:
Verify that something is being received in **kwargs, if not received anything then it has to be removed.